app/models/event.rb

```diff
@@ -35,8 +35,6 @@ class Event < ActiveRecord::Base
   validates :program, presence: true
   validates :max_attendees, numericality: { only_integer: true, greater_than_or_equal_to: 1, allow_nil: true }
 
-  validate :max_attendees_no_more_than_room_size
```

Comment:
The problem with removing this from here is that when a proposal is scheduled in a room that can have up to 10 people, when the submitter edits the proposal, can set `max_attendees` to 15, and enable require_registration. Which will allow 15 people to register to a session that takes place in a room that can only hold only 10 people.

Now of course the problem can work the other way around, which is what I assume you are trying to fix, ie. what if we try to schedule a talk with 15 registered attendees in a room that can only have 10 seats. That's indeed an issue, one we are aware of and trying to resolve. But in my opinion:
1) we should not break scheduling because of this. An organizer wants to schedule a specific session into a specific room at a specific time, a warning would be excellent, but disallowing it all together, I don't know. Mainly because the organizer cannot properly remedy this. Once 15 people have registered to attend, the organizer cannot delete the registrations.

2) the workflow of max_attendees is a subject for the proposal registration process to solve, not this PR. Unless there is something really failing with your changes. Is there?